azurerm/internal/services/keyvault/key_vault_resource.go

```diff
@@ -73,6 +73,7 @@ func resourceArmKeyVault() *schema.Resource {
 					string(keyvault.Standard),
 					string(keyvault.Premium),
 				}, false),
+				DiffSuppressFunc: suppress.CaseDifference,
```

Comment:
Could we have a link to an bug/issue here that describes why we need to do this?